documentation/contributing/pullrequests.markdown

```diff
@@ -56,16 +54,18 @@ The ``*-devel`` branches must not be force-pushed.
 No one is allowed to directly commit to the ``*-devel`` branches of the repositories. Instead they should create feature branches and add pull-requests.
 The only commits that might be pushed directly are cherry-picks from older branches (see below) and administrative changes (e.g. CHANGELOGS, tags).
 
-### 1+1 == Merge
+### Two "Approves" == Merge
 
-If you see a trustworthy "+1" note in a request or the requestor is a fellow maintainer and all feedback has been addressed by the requestor,
-merge the request after your own review. Otherwise add a "+1" note after you are satisfied after the review.
+If you see a trustworthy approval review in a request or the requestor is a fellow maintainer and all feedback has been addressed by the requestor,
+merge the request after your own review. Otherwise submit an "Approval" review after you are satisfied after the review.
 
 ### Squash-Merge If Possible
 
 Pull-requests with a single commit should *always* be squash-merged (github supports this via a drop-down list on the "merge" button).
 If a request contains "fixup commits" it should also be squash-merged. If you are unsure this is to the requestor's liking, ask him.
 
+Note: Github recently added a [Rebase and merge](https://github.com/blog/2243-rebase-and-merge-pull-requests) option for approving pull requests - this is likely the recommended method going forward, but this has not been finalized.
+
```

Comment:
So is the question whether we'll accept `rebase-and-merge` as another option for merging PRs or not? If so here's my 2cents.

Finally I'm getting better idea about this new feature on Github (in yesterday's meeting I wasn't sure yet about what it does). I personally prefer a merge commit to the rebased-and-merge as of now. Reason is pretty much explained by @v4hn above but let me just add:

- Looks like from @v4hn's experiment, the commits that are merged by `rebase-and-merge` don't seem to retain info of which PR they are suggested and got merged in (e.g. [this commit](https://github.com/v4hn/foo/commit/9036bfe2b8bb415705a7d3004609944fc061a14d) was suggested in [this PR](https://github.com/v4hn/foo/pull/1) but in the commit page I don't see any info for that). Discussion at the PR pages is a great source of info in many ways, and it's very nice of Github currently to indicate on each commit page the link to the associated PR.
- That said, if Github in the future changes the rebase-and-merge's behavior to enable us to track down the relevant PR from commits, I would likely to prefer `rebase-and-merge` to `merge-commit`.